### PR TITLE
remove nftables, iptables and ip6tables from image

### PR DESCRIPTION
**Pull request by edcdavid**
removing iptables and nftables from partner image to prevent configuration of extra rules

### CODE REVIEW

File: test-partner/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM registry.access.redhat.com/ubi8/ubi:latest
 
 RUN dnf install -y wget make hostname iproute iputils openssh openssh-clients podman psmisc
-
+RUN dnf remove -y iptables ip6tables nftables
```
Comment (rdavid) [inline test-partner/Dockerfile:4]: podman depends on and installs iptables ip6tables nftables. Do we use podman in the image? If not remove podman from the installation list, firewall utilities will not be installed.

Comment (sebrandon1) [reply test-partner/Dockerfile:4]: I believe we need `podman` installed for our `fsdiff` test in the `cnf-certification-test` repo.

**Review by sebrandon1 — APPROVED** [review]

File: test-partner/Dockerfile (hunk 1 above)
Comment (edcdavid) [reply test-partner/Dockerfile:4]: ok thanks for pointing this. I will remove the PR

**Comment by edcdavid** [comment]
should not be needed to pass the nftables/iptables test closing for now

**Pull request closed without merging**